Source/Core/VideoBackends/Vulkan/FramebufferManager.h

```diff
@@ -160,7 +157,7 @@ class FramebufferManager : public FramebufferManagerBase
   std::unique_ptr<StreamBuffer> m_poke_vertex_stream_buffer;
   std::vector<EFBPokeVertex> m_color_poke_vertices;
   std::vector<EFBPokeVertex> m_depth_poke_vertices;
-  VkPrimitiveTopology m_poke_primitive_topology = VK_PRIMITIVE_TOPOLOGY_TRIANGLE_LIST;
+  PrimitiveType m_poke_primitive = PrimitiveType::Points;
```

Comment:
Intentional?

Reply:
Nope, but it shouldn't have any effect as it's overwritten anyway. I'll change it for consistency.